executor/executor.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"os/exec"
 	"syscall"
+  "io/ioutil"
```

Comment:
Please make sure you have `gofmt` installed as it will auto-indent for you on save.